client/login/wp-login/style.scss

```diff
@@ -186,7 +186,12 @@ $image-height: 47px;
 .layout.is-jetpack-woocommerce-flow {
 	background-color: var( --color-woocommerce-onboarding-background );
 
+	.wp-login__links {
+		margin-top: 1em;
+	}
+
 	.wp-login__links a {
 		border-bottom: none;
+		line-height: 2.5em;
```

Comment:
These `line-height` and `margin-top` seem _different_ from just changing colours like the other `is-jetpack-login`/`is-jetpack-woocommerce-flow` stuff seems to be. Should they be applied to all variations of the login page instead?

I didn't have a look how they actually change the layout and I appreciate if they're brand specific of course. :-) Just presense of these might surprise folks doing changes in generic layout.

Reply:
I added this just to tighten up the footer a bit, because it felt like there was too much spacing without the border separating it like the normal Calypso flow does. It didn't come from our design/brand guidelines, but the area looked off to me.

Before:

<img width="426" alt="Screen Shot 2019-05-16 at 9 05 05 AM" src="https://user-images.githubusercontent.com/689165/57855889-bb819e80-77b9-11e9-9d54-6b8b9929ca11.png">

After:

<img width="469" alt="Screen Shot 2019-05-16 at 9 03 57 AM" src="https://user-images.githubusercontent.com/689165/57855898-c3414300-77b9-11e9-9178-7daaf5b7c95e.png">

Reply:
Oh right, makes sense!

I don't know if more generic way to implement that would be more clarity, or maybe just an inline code comment would suffice? This totally isn't blocking comment, feel free to ship without any changes. I'm just noting what seemed confusing to me.

I think the idea of 4em line-height was that it follows material patterns of 56px space for "a single line list item that has an icon".

https://material.io/design/components/lists.html#specs

Seems like 48px would be good as well if there's no icon but no idea how that translates to `em` value. We have this now but I don't know how to use it:

https://github.com/Automattic/wp-calypso/blob/641ebd2a0061877ca281dd52e59d7ac9f0a1ce76/packages/muriel-style/typography.scss

cc @griffbrad @blowery :wave: might know.

...maybe this list view isn't following Material/Muriel patterns at all and this isn't relevant. :-)

Reply:
Good call - we do want Muriel patterns. I hadn't seen anything in the Muriel guide lines about line-height, so thanks for the reminder about looking at Material :).

I went ahead and adjusted this to 3.4em (which seems like the correct conversion at font-size 14) for now. While not as big as a difference, it's still looking better to me.

We also plan on doing a design sweep of these PRs in context with the whole onboarding flow (part of which will be self hosted for the WooCommerce steps), so we can make more updates in the future if needed.